gammapy/time/plot.py

```diff
@@ -2,31 +2,126 @@
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
 
-__all__ = ['plot_time_difference_distribution',
+from ..datasets import fetch_fermi_catalog
+
+__all__ = ['plot_light_curve',
            ]
 
 
-def plot_time_difference_distribution(time, ax=None):
-    """Plot event time difference distribution.
+def plot_light_curve(name_3FGL, fermi_met_start = 2.3933e8, fermi_met_end = 3.65467550e8):
+    """Plot flux as a function of time for a fermi 3FGL object.
 
     Parameters
     ----------
-    time : `~astropy.time.Time`
-        Event times (must be sorted)
-    ax : `~matplotlib.axes.Axes` or None
-        Axes
-
-    Returns
-    -------
-    ax : `~matplotlib.axes.Axes`
-        Axes
+    name_3FGL : `string`
+        The 3FGL catalog name of the object to plot
+    fermi_met_start : `int`
+        Start time of the light curve in Fermi MET
+    fermi_met_end : `int`
+        End time of the light curve in Fermi MET
+
+    Usage
+    gammapy.time.plot_light_curve('3FGL J0349.9-2102', 2.1e8, 3.2e8)
```

Comment:
You should add a full example to the docstring.
This means adding a section:
```
Examples
-------------
```
and a plot directive as shown e.g. in this example: https://gammapy.readthedocs.org/en/latest/api/gammapy.irf.EffectiveAreaTable.html

So Sphinx will actually execute that code and put the image in the docs.
Fetching lots of datasets in the docs build will become problematic, but for now I think it's OK.